### DISCUSSION TITLE
Add a way to change output folder

### DISCUSSION THREAD

**Discussion by Buppsie** [Official-Husko/NN-Downloader#26] [Ideas]
add someway (config file?) to change the out put folder for the downloaded images

**Comment by Official-Husko** [comment]
Do you mean change the overall saving directory or only for certain items which the user is downloading? Can you elaborate a bit further?

**Reply by Buppsie** [reply]
the overall saving directory

**Reply by Official-Husko** [reply]
Sure i can look into that. Just note that config files and databases will stay in the main program folder to keep it all in one place.

**Reply by Official-Husko** [reply]
Just a quick update, i haven't forgotten about this but I'm currently very busy with quite a few things with one being the rewritting of another tool of mine to a more clean and maintainable codebase. I will tackle this issue as soon as i got time.
